### PR TITLE
Fix navigate to download page on download notification click

### PR DESCRIPTION
**Pull request by NavayuvanSB**
### Changes done
- Downloads page will open on clicking the download notification

### CODE REVIEW

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt [hunk 1]
```diff
@@ -51,11 +59,13 @@ class VideoDownloadService : DownloadService(
     }
 
     override fun getForegroundNotification(downloads: MutableList<Download>): Notification {
-        // TODO : Add intent and also download message
+
+        val notifyPendingIntent = getNotifyPendingIntent()
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:63]: I cannot understand what this variable refers to by seeing its name. Please change it to meaningful name.
Name should explain its intent

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt [hunk 2]
```diff
@@ -51,11 +59,13 @@ class VideoDownloadService : DownloadService(
...
+
         return notificationHelper.buildProgressNotification(
                 applicationContext,
                 R.drawable.ic_download,
-                null,
+                notifyPendingIntent,
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:68]:
```suggestion
                getNotifyPendingIntent(),
```

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt [hunk 3]
```diff
@@ -83,9 +93,21 @@ class VideoDownloadService : DownloadService(
             Download.STATE_FAILED -> notification = getFailedNotification()
             Download.STATE_REMOVING -> downloadMonitor.deleteVideo(download)
         }
+
+        notification?.contentIntent = getNotifyPendingIntent()
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:97]: I could not understand what this variable refers to by seeing its name. Please change it to meaningful name.
Name should explain its intent

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt [hunk 4]
```diff
@@ -83,9 +93,21 @@ class VideoDownloadService : DownloadService(
...
         NotificationUtil.setNotification(this, nextNotificationId, notification)
     }
 
+    private fun getNotifyPendingIntent(): PendingIntent{
+        val notifyIntent = Intent(this, DownloadsActivity::class.java).apply {
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:102]: notifyIntent variable does not conveying any meaning, please change it to meaningful name

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt [hunk 5]
```diff
@@ -83,9 +93,21 @@ class VideoDownloadService : DownloadService(
...
+            flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
+        }
+        val notifyPendingIntent = PendingIntent.getActivity(
```
Comment (Karthik-0) [inline course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:105]: return this directly

**Review by Karthik-0 — CHANGES_REQUESTED** [review]

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt (hunk 2 above)
Comment (NavayuvanSB) [reply course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:68]: Okay @Karthik-0 , will change that.

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt (hunk 3 above)
Comment (NavayuvanSB) [reply course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:97]: Okay @Karthik-0, will change that.

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt (hunk 4 above)
Comment (NavayuvanSB) [reply course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:102]: Okay @Karthik-0, will change that.

File: course/src/main/java/in/testpress/course/services/VideoDownloadService.kt (hunk 5 above)
Comment (NavayuvanSB) [reply course/src/main/java/in/testpress/course/services/VideoDownloadService.kt:105]: Okay @Karthik-0, will change that.

**Comment by NavayuvanSB** [comment]
Made the changes @Karthik-0, please review it.

**Review by Karthik-0 — APPROVED** [review]

**Pull request merged**